web/controllers/days_controller.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Janitor.DaysController do
+  use Janitor.Web, :controller
+  import Janitor.DaysQueryHelpers
+  alias Janitor.Repo
+
+  def index(conn, %{month: month}) do 
```

Comment:
Please use string keys when interacting with user input. External data can’t safely be converted to atoms, because the atom table isn’t garbage-collected. Run out of atoms (e.g. via a DDoS or just a very long running process) and you OOM  the app.